web/src/stream_list.ts

```diff
@@ -839,6 +841,31 @@ export function set_event_handlers({
 }: {
     on_stream_click: (stream_id: number, trigger: string) => void;
 }): void {
+    $("#stream_filters").on(
+        "mouseenter",
+        "li .subscription_block .stream-name",
+        function (this: HTMLElement, e) {
+            const stream_id = stream_id_for_elt($(e.target).parents("li"));
+            const current_narrow_stream_id = narrow_state.stream_id();
+            const current_topic = narrow_state.topic();
+            if (!(current_narrow_stream_id === stream_id && current_topic)) {
+                return;
+            }
+            e.stopPropagation();
+            const $elem = $(this);
+            tippy.default(util.the($elem), {
+                showOnCreate: true,
+                onShow(instance) {
+                    instance.setContent($t({defaultMessage: "Go to channel feed"}));
```

Comment:
We should probably advertise the `s` keyboard shortcut for switching to the channel feed here, right?

Reply:
Good idea, added that :)